lib/npm/install.js

```diff
@@ -18,6 +18,8 @@ function install(context, next) {
   var bailed = false;
   context.emit('data', 'info', context.module.name + ' npm:', 'npm install started');
 
+  context.emit('data', 'verbose', context.module.name + ' tmp', 'Using --tmpdir="' + options.env['npm_config_tmp'] + '"');
```

Comment:
Why `using --tmpdir`? This isn't a citgm option is it? If you can do `npm --tmpdir` then I guess change it to `Using npm --tmpdir="`.